### PR TITLE
Pass params

### PR DESCRIPTION
**Pull request by Kataderon**

### CODE REVIEW

File: jobs/Scripts/simpleRender.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def createArgsParser():
 	parser.add_argument('--testCases', required=True)
 	parser.add_argument('--SPU', required=False, default=10)
 	parser.add_argument('--fail_count', required=False, default=0, type=int)
+	parser.add_argument('--threshold', required=True)
```
Comment (MagisterDemens) [inline jobs/Scripts/simpleRender.py]: Why required=True ?

**Comment by Kataderon** [comment]
https://rpr.cis.luxoft.com/job/mayaPassParams/6/

**Comment by Kataderon** [comment]
New report:
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/mayaPassParams/8/Test_20Report/

**Comment by Kataderon** [comment]
Problems with osx

**Comment by Kataderon** [comment]
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/mayaPassParams/9/Test_20Report/

File: jobs/Scripts/base_functions.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def get_scene_name():
 
 def skipped_case_report(case):
```
Comment (MagisterDemens) [inline jobs/Scripts/base_functions.py]: skipped, error, pass -> case['status']

File: jobs/Scripts/extensions/Uber.py [hunk 3]
```diff
@@ -83,4 +83,28 @@ def resetAttribute():
...
+	cmds.connectAttr(texture + '.repeatUV', file + '.repeatUV', f=True)
+	cmds.connectAttr(texture + '.offset', file + '.offset', f=True)
+	cmds.connectAttr(texture + '.rotateUV', file + '.rotateUV', f=True)
+	cmds.connectAttr(texture + '.noiseUV', file + '.noiseUV', f=True)
+	cmds.connectAttr(texture + '.vertexUvTwo', file + '.vertexUvTwo' , f=True)
+	cmds.connectAttr(texture + '.vertexUvThree', file + '.vertexUvThree', f=True)
+	cmds.connectAttr(texture + '.vertexCameraOne', file + '.vertexCameraOne', f=True)
+	cmds.connectAttr(texture + '.outUV', file + '.uv', f=True)
+	cmds.connectAttr(texture + '.outUvFilterSize', file + '.uvFilterSize')
+	cmds.connectAttr(texture + '.vertexUvOne', file + '.vertexUvOne')
```
Comment (MagisterDemens) [inline jobs/Scripts/extensions/Uber.py]: why f=False / True

File: jobs/Scripts/simpleRender.py [hunk 4]
```diff
@@ -247,8 +256,8 @@ def main(args):
 			window_titles = get_windows_titles()
 			error_window = set(fatal_errors_titles).intersection(window_titles)
 			if error_window:
-				core_config.main_logger.error("Error window found: {}".format(error_window))
-				core_config.main_logger.error("Found windows: {}".format(window_titles))
+				core_config.main_logger.info('Error window found: {}'.format(error_window))
+				core_config.main_logger.info('Found windows: {}'.format(window_titles))
```
Comment (MagisterDemens) [inline jobs/Scripts/simpleRender.py]: min WARGNING

File: jobs/Scripts/extensions/Uber.py (hunk 3 above)
Comment (Kataderon) [reply jobs/Scripts/extensions/Uber.py]: Forces the connection. If the destination is already connected, the old connection is broken and the new one made.
--


Don't know why

Comment (Kataderon) [reply jobs/Scripts/extensions/Uber.py]: Maya generates this

**Comment by Kataderon** [comment]
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/mayaPassParams/12/Test_20Report/
errors on broken Rome

**Pull request merged**